tsconfig-strict.json

```diff
@@ -44,6 +44,7 @@
     "core/templates/domain/utilities/browser-checker.service.spec.ts",
     "core/templates/expressions/expression-parser.service.ts",
     "core/templates/expressions/expression-parser.service.spec.ts",
+    "core/templates/filters/string-utility-filters/normalize-whitespace-punctuation-and-case.pipe.ts",
```

Comment:
Please put this in in the alphabetical order, so below `"core/templates/filters/string-utility-filters/camel-case-to-hyphens.pipe.spec.ts"`.

Reply:
Done!